manager/allocator/cnmallocator/networkallocator.go

```diff
@@ -94,6 +94,9 @@ type NetworkConfig struct {
 	// SubnetSize specifies the subnet size of the networks created from
 	// the default subnet pool
 	SubnetSize uint32
+
+	// VxlanUDPPort specifies VxLAN UDP port number for VxLAN traffic
+	VxlanUDPPort uint32
```

Comment:
Since we've moved the call to set the Vxlan port, does this field still need to be here?

Reply:
Why is this needed here?